simulator/mips/t/unit_test.cpp

```diff
@@ -702,6 +698,91 @@ TEST_CASE( "MIPS64_instr: dclz 0x02ffaa00cc720000")
 }
 ////////////////////////////////////////////////////////////////////////////////
 
+TEST_CASE( "MIPS32_instr: div 1 by 1")
+{
+    CHECK(MIPS32Instr(0x0229001a).get_disasm() == "div $s1, $t1");
+    CHECK(MIPS32Instr(0x0229001a).is_divmult());
+    CHECK(MIPS32Instr("div").is_divmult());
+
+    MIPS32Instr instr( "div");
+    instr.set_v_src( 1, 0);
+    instr.set_v_src( 1, 1);
+    instr.execute();
+    CHECK( instr.get_v_dst2() == 0);
+    CHECK( instr.get_v_dst()  == 1);
+}
+
+TEST_CASE( "MIPS32_instr: div -1 by 1")
+{
+    MIPS32Instr instr( "div");
+    instr.set_v_src( 0xffffffff, 0);
+    instr.set_v_src( 1, 1);
+    instr.execute();
+    CHECK( instr.get_v_dst2() == 0);
+    CHECK( instr.get_v_dst()  == 0xffffffff);
+}
+
+TEST_CASE( "MIPS32_instr: div -1 by -1")
+{
+    MIPS32Instr instr( "div");
+    instr.set_v_src( 0xffffffff, 0);
+    instr.set_v_src( 0xffffffff, 1);
+    instr.execute();
+    CHECK( instr.get_v_dst2() == 0);
+    CHECK( instr.get_v_dst()  == 1);
+}
+
+TEST_CASE( "MIPS32_instr: div 1 by -1")
+{
+    MIPS32Instr instr( "div");
+    instr.set_v_src( 1, 0);
+    instr.set_v_src( 0xffffffff, 1);
+    instr.execute();
+    CHECK( instr.get_v_dst2() == 0);
+    CHECK( instr.get_v_dst()  == 0xffffffff);
+}
+
+TEST_CASE( "MIPS32_instr: div 0 by 1")
+{
+    MIPS32Instr instr( "div");
+    instr.set_v_src( 0, 0);
+    instr.set_v_src( 1, 1);
+    instr.execute();
+    CHECK( instr.get_v_dst2() == 0);
+    CHECK( instr.get_v_dst()  == 0);
+}
+
+TEST_CASE( "MIPS32_instr: div 1 by 0")
+{
+    MIPS32Instr instr( "div");
+    instr.set_v_src( 1, 0);
+    instr.set_v_src( 0, 1);
+    instr.execute();
+    CHECK( instr.get_v_dst2() == 0);
+    CHECK( instr.get_v_dst()  == 0);
```

Comment:
This should be converted to a stub for #130 and commented out, likewise other items:
```diff
-     CHECK( instr.get_v_dst2() == 0);
-     CHECK( instr.get_v_dst()  == 0);
+     CHECK( instr.trap() != Trap::NO_TRAP);
```

Reply:
So, looking at the latest comment, there's no need to replace these `CHECKs` with NO_TRAP `CHECK`?

Reply:
Yes.